pypeit/sensfunc.py

```diff
@@ -264,10 +264,11 @@ def splice(self, wave):
         # Interpolate over gaps
         zeros = sensfunc_splice == 0.
         if np.any(zeros):
-            msgs.info("Interpolating over gaps")
+            msgs.info("Interpolating over gaps (and extrapolating with fill_value=1, if need be)")
             interp_func = scipy.interpolate.interp1d(wave_splice[np.invert(zeros)],
                                                  sensfunc_splice[np.invert(zeros)],
-                                                 kind='nearest', fill_value='extrapoloate')
+                                                 kind='nearest', fill_value=0., bounds_error=False) #
```

Comment:
According to the documentation fill_value='extrapolate' will extrapolate outside the boundaries and will not raise a bounds_error. I don't understand why you have changed this to replace with fill_value=0.0. If there is a bug, it may be that we need to set assume_sorted=True here after we slice to the boolean, but I was under the impression that this would produce a sorted array if the input array is sorted.

Reply:
Yes, that is what the docs say but it crashed anyways.

Because this crash occurs after ~1 hour of telluric optimization
it is impossible to properly debug.